cmd/fs-v1-multipart.go

```diff
@@ -361,9 +361,9 @@ func (fs fsObjects) PutObjectPart(bucket, object, uploadID string, partID int, s
 		return "", toObjectErr(err, minioMetaMultipartBucket, uploadIDPath)
 	}
 
+	// Append the part in background.
+	errCh := fs.bgAppend.append(fs.storage, bucket, object, uploadID, fsMeta)
```

Comment:
Yes, you are right. It wouldn't block for the reason I stated. I failed to notice the send on errCh.